contrib/zpages/src/main/java/io/opencensus/contrib/zpages/RpczZPageHandler.java

```diff
@@ -233,6 +233,11 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
     out.write("<meta charset=\"utf-8\">\n");
     out.write("<title>RpcZ</title>\n");
     out.write("<link rel=\"shortcut icon\" href=\"//www.opencensus.io/favicon.ico\"/>\n");
+    out.write(
```

Comment:
Consider putting the style code into another static method (like `emitStyle()`), so that we can find and update the styles more quickly in the future.